### PR TITLE
added trio.md to content/projects/

### PR DESCRIPTION
**Pull request by jeffschwartz**
Added trio.md to content/projects/

### CODE REVIEW

**Comment by erquhart** [comment]
This PR is failing tests because your fork needs to be updated, its a year old.

**Comment by jeffschwartz** [comment]
I'm sorry but I don't understand how the repo I forked can be behind. There is only the one branch, master, and I did a fetch from upstream:
 
Screen Shot 2019-12-03 at 3 03 15 PM

and nothing is stale on my fork.

Sorry to be pain but what am I missing here.

**Comment by erquhart** [comment]
You fetched but you didn’t actually pull the fetched changes into your branch. With master checked out, and assuming your PR isn’t from your master branch, run:

```
git pull upstream master
```

Then rebase your feature branch onto master with the newly updated master.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jeffschwartz** [comment]
Thank you, @erquhart, for responding. I started over with a brand new fork, cloned it locally and configured remote to be as shown below:

Screen Shot 2019-12-04 at 10 56 39 AM

Then I ran `git pull upstream master` as you requested:

Screen Shot 2019-12-04 at 10 43 53 AM

As the above image shows, the master branch is already up to date. FWIW this is the exact same procedure I ran for my first pull request. 

If I would now branch off of local master to create my feature branch `feature-trio`, and add my content to it, and then issue a pull request from that then the results will be exactly the same as my first pull request.

__To summarize the problem, pulling from `upstream master` is not finding any changes to be applied to my local master branch.__

**Comment by jeffschwartz** [comment]
I create a brand new fork, and I think I did the exact same steps as the last time but now it has passed all the tests. Obviously I screwed up the 1st time lol and I'm sorry that I took time out of your busy day to have to deal with it. Thank you, @erquhart , for your patience.
